src/modules/Layout/Header/Header.jsx

```diff
@@ -16,9 +15,9 @@ const Header = ({ children }) => {
       </div>
       <div className={b('col', { 'middle': true })}>
         <Container>
-          <Nav>
+          <div className="j-header__nav">
```

Comment:
I think header should know nothing about navigation - it's like layout component. But maybe i'm wrong